patterns/atoms/global/colors/_colors.scss

```diff
@@ -0,0 +1,28 @@
+
+.colors__color1 {
```

Comment:
The theory is that when you import this pattern, you would use your own data to give the pattern it’s colors. So, instead of pulling in colors.scss you could be using whatever color selection you’d like. Then you can give this pattern's classnames a color (.colors_color1 {background-color: $primary-color;}) or you could give the pattern a different set of classNames to match your project’s colors:
data:
  colors:
    color1:
      class: primary__color
      text: Primary Color
    color2:
      class: secondary__color
      text: Secondary
    color3:
      class: tertiary__color
      text: Tertiary